src/main/java/com/blackducksoftware/integration/jira/config/controller/action/ConfigActions.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Black Duck JIRA Plugin
+ *
+ * Copyright (C) 2019 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.blackducksoftware.integration.jira.config.controller.action;
+
+import com.atlassian.sal.api.pluginsettings.PluginSettingsFactory;
+import com.blackducksoftware.integration.jira.common.PluginSettingsWrapper;
+
+public abstract class ConfigActions {
```

Comment:
It's nice to have parent classes that provide us with extra functionality, but should we be favoring composition over inheritance for things like this? As far as I saw, we weren't using ConfigActions anywhere and only used it as a way to cut out duplicate code.